src/analytics/events.js

```diff
@@ -0,0 +1,9 @@
+import { Event } from './GoogleAnalytics';
+
+export function CopyTextAttribution(text) {
+  return new Event('Attribution', 'Copy Text', text);
```

Comment:
could we log the anonymized IP here?

Reply:
Why log the anonymized IP?

Reply:
To keep track of how many users are using the copy attribution feature (is one user using it a lot or is it a bunch of different users?). Google Analytics might support this out of the box though, so I'm not sure if we need to log it here.

Reply:
I'm not sure whether events are individualized or not on GA. My first guess would be that they're not.